zio-kafka-test/src/test/scala/zio/kafka/OOMSpecXmx300m.scala

```diff
@@ -0,0 +1,88 @@
+package zio.kafka
+
+import org.apache.kafka.clients.producer.ProducerRecord
+import zio.ZIO
+import org.apache.kafka.common.KafkaException
+import zio.kafka.consumer.{ Consumer, Subscription }
+import zio.kafka.embedded.Kafka
+import zio.kafka.producer.Producer
+import zio.kafka.serde.Serde
+import zio.test.Assertion._
+import zio.test.TestAspect._
+import zio.test._
+
+/**
+ * This test checks the fix for the issue https://issues.apache.org/jira/browse/KAFKA-4090
+ */
+object OOMSpecXmx300m extends ZIOSpecWithSslKafka {
+
+  override val kafkaPrefix: String = "oom-spec"
+  override def spec: Spec[Kafka, Any] =
+    suite("OOM check")(
```

Comment:
We're not testing for OOM so this name is misleading.
```suggestion
    suite("Plain connection on TLS port")(
```